integration-cli/docker_api_exec_resize_test.go

```diff
@@ -78,9 +78,9 @@ func (s *DockerSuite) TestExecResizeImmediatelyAfterExecStart(c *check.C) {
 
 		if err == nil {
 			rc.Close()
+		}  else {
```

Comment:
We should combine this with the checks above, and use `err != nil`, which is a more common approach;

```go
_, rc, err := request.Post(fmt.Sprintf("/exec/%s/resize?h=24&w=80", execID), request.ContentType("text/plain"))
// It's probably a panic of the daemon if io.ErrUnexpectedEOF is returned.
if err != nil {
	if err == io.ErrUnexpectedEOF {
		return fmt.Errorf("The daemon might have crashed.")
	}
	return errors.Wrap(err, "failed to exec resize")
}

rc.Close()

// We only interested in the io.ErrUnexpectedEOF error, so we return nil otherwise.
return nil
```

Reading the comment though, it looks like previously we ignored other errors; is there a specific error we don't want to ignore?

Reply:
I think the exec start/resize issue should be detected by this test case if we do not ignore those errors.
I run several rounds of test, so far no strange error found. I would run more round of tests to see how stable this test would be.

Reply:
Changes made. But I dose not use the `errors` package because all other errors is generated by `fmt.Errorf`, kind of  strange to change style... I will make the change if you still think it is necessary. @thaJeztah